### PR TITLE
Make TRANS versionIRI resolvable

### PR DESCRIPTION
**Pull request by allenbaron**
To fix the versionIRI resolvability error of the Pathogen Transmission Ontology on the OBO dashboard.

### CODE REVIEW

File: config/trans.yml [hunk 1]
```diff
@@ -12,3 +12,11 @@ products:
...
 example_terms:
 - TRANS_0000000
+
+- regex: ^/obo/trans/releases/([^\s/]+)/(\S+)$
+  replacement: https://raw.githubusercontent.com/DiseaseOntology/PathogenTransmissionOntology/v$1/src/ontology/$2
+  tests:
+  - from: /releases/2022-10-10/trans.owl
+    to: https://raw.githubusercontent.com/DiseaseOntology/PathogenTransmissionOntology/v2022-10-10/src/ontology/trans.owl
+  - from: /releases/2022-10-10/trans.obo
+    to: https://raw.githubusercontent.com/DiseaseOntology/SymptomOntology/v2022-10-10/src/ontology/trans.obo
```
Comment (jamesaoverton) [inline config/trans.yml:22]: Should this one be `SymptomOntology`?

**Comment by allenbaron** [comment]
Hahaha.... no. Sorry about that. I was lazy and just copied it from the Symptom Ontology. I guess I missed that one.

File: config/trans.yml [hunk 2]
```diff
@@ -12,3 +12,11 @@ products:
 term_browser: ontobee
 example_terms:
 - TRANS_0000000
+
```
Comment (jamesaoverton) [inline config/trans.yml:15]: Missing an `entries:` line before the first `regex` entry.

**Comment by jamesaoverton** [comment]
Thanks. I prefer to avoid `regex` because it can be a footgun, but it makes sense given the layout of the GitHub repository in this case, and there are tests to verify that it's working as expected.

**Pull request merged**